keras/models.py

```diff
@@ -819,36 +819,52 @@ def compile(self, optimizer, loss,
     def fit(self, x, y, batch_size=32, epochs=10, verbose=1, callbacks=None,
             validation_split=0., validation_data=None, shuffle=True,
             class_weight=None, sample_weight=None, initial_epoch=0, **kwargs):
-        """Trains the model for a fixed number of epochs.
+        """Trains the model for a fixed number of epochs (iterations on a dataset).
 
         # Arguments
-            x: input data, as a Numpy array or list of Numpy arrays
-                (if the model has multiple inputs).
-            y: labels, as a Numpy array.
-            batch_size: integer. Number of samples per gradient update.
-            epochs: integer. Number of epochs to train the model.
+            x: Numpy array of training data, or 1-element list of Numpy arrays.
```

Comment:
"or 1-element list of Numpy arrays" does not need to be specified, since this is not how we want people to use `fit` on `Sequential`.